src/app/header/header.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-default">
```

Comment:
header component is not part of loginAndRegister feature. Should be commited to different branch.